package.json

```diff
@@ -31,7 +31,6 @@
     "@sphereon/ssi-sdk-lto-did-provider": "0.8.1-unstable.86",
     "@sphereon/ssi-sdk-vc-api-issuer": "0.8.1-unstable.86",
     "@sphereon/ssi-types": "0.8.1-unstable.86",
-    "@unimodules/core": "^7.1.2",
```

Comment:
AFAIK, we explicitly add this for OpenID4VCI support, which is vital in the app. Have you tested the app to be working with one of the issuers from the README?

Reply:
Seems this package is deprecated

[npm: @unimodules/core](https://www.npmjs.com/package/@unimodules/core)

And has been replaced by the expo package

https://blog.expo.dev/whats-new-in-expo-modules-infrastructure-7a7cdda81ebc

So i think we need to look into replacing it.

Reply:
The ios compiler errors out because duplicate symbols when unimodules is in the dependencies, because it's already present by default in the expo package. Not sure if OpenID4VCI will still be able to import it, or that is has stopped working now.

Reply:
Okay let's see how this plays out

Reply:
There is still an issue met the patches